lib/astarte_core/triggers/policy_protobuf.ex

```diff
@@ -0,0 +1,36 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2017 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Core.Triggers.PolicyProtobuf do
+  @external_resource Path.expand("/policy_protobuf", __DIR__)
+
+  use Protobuf, from: Path.wildcard(Path.expand("policy_protobuf/*.proto", __DIR__))
+
+  @doc """
+  Utility macro, `use Astarte.Core.Triggers.PolicyProtobuf` injects aliases
+  for all the structs defined in the `Astarte.Core.Triggers.PolicyProtobuf`
+  namespace
+  """
+  defmacro __using__(_opts) do
+    for {{:msg, msg_module}, _} <- __MODULE__.defs() do
+      quote do
+        alias unquote(msg_module)
+      end
+    end
+  end
```

Comment:
This snippet was me discovering macros for the first time and doing cursed stuff with them. I don't know if it's a good idea keeping this in new code vs just explicitly adding aliases where needed.

Reply:
`use Astarte.Core.Triggers.PolicyProtobuf` was pretty much unused, so no need for that macro